templates/index.ts

```diff
@@ -23,3 +23,4 @@ export { default as uptimekuma } from "./uptimekuma";
 export { default as wikijs } from "./wikijs";
 export { default as wordpress } from "./wordpress";
 export { default as whiteboard } from "./whiteboard";
+export { default as imgproxy } from "./imgproxy";
```

Comment:
The imports should be ordered alphabetically. I have an option in VS Code that organizes my imports automatically.